public/components/model_group/__tests__/model_group.test.tsx

```diff
@@ -25,37 +25,45 @@ const setup = () => {
 };
 
 describe('<ModelGroup />', () => {
-  it('should display model name, action buttons, overview-card, tabs and tabpanel after data loaded', async () => {
-    setup();
-
-    await waitFor(() => {
-      expect(screen.queryByTestId('model-group-loading-indicator')).toBeNull();
-    });
-    expect(screen.getByText('model1')).toBeInTheDocument();
-    expect(screen.getByText('Delete')).toBeInTheDocument();
-    expect(screen.getByText('Register version')).toBeInTheDocument();
-    expect(screen.queryByTestId('model-group-overview-card')).toBeInTheDocument();
-    expect(screen.queryByTestId('model-group-overview-card')).toBeInTheDocument();
-    expect(screen.queryByTestId('model-group-overview-card')).toBeInTheDocument();
-    expect(screen.getByRole('tab', { name: 'Versions' })).toHaveClass('euiTab-isSelected');
-    expect(within(screen.getByRole('tabpanel')).getByText('Versions')).toBeInTheDocument();
-  });
-
-  it('should display consistent tabs content after tab clicked', async () => {
-    setup();
-
-    await waitFor(() => {
-      expect(screen.queryByTestId('model-group-loading-indicator')).toBeNull();
-    });
-    expect(screen.getByRole('tab', { name: 'Versions' })).toHaveClass('euiTab-isSelected');
-    expect(within(screen.getByRole('tabpanel')).getByText('Versions')).toBeInTheDocument();
-
-    await userEvent.click(screen.getByRole('tab', { name: 'Details' }));
-    expect(screen.getByRole('tab', { name: 'Details' })).toHaveClass('euiTab-isSelected');
-    expect(within(screen.getByRole('tabpanel')).getByText('Details')).toBeInTheDocument();
-
-    await userEvent.click(screen.getByRole('tab', { name: 'Tags' }));
-    expect(screen.getByRole('tab', { name: 'Tags' })).toHaveClass('euiTab-isSelected');
-    expect(within(screen.getByRole('tabpanel')).getByText('Tags')).toBeInTheDocument();
-  });
+  it(
+    'should display model name, action buttons, overview-card, tabs and tabpanel after data loaded',
+    async () => {
+      setup();
+
+      await waitFor(() => {
+        expect(screen.queryByTestId('model-group-loading-indicator')).toBeNull();
+      });
+      expect(screen.getByText('model1')).toBeInTheDocument();
+      expect(screen.getByText('Delete')).toBeInTheDocument();
+      expect(screen.getByText('Register version')).toBeInTheDocument();
+      expect(screen.queryByTestId('model-group-overview-card')).toBeInTheDocument();
+      expect(screen.queryByTestId('model-group-overview-card')).toBeInTheDocument();
+      expect(screen.queryByTestId('model-group-overview-card')).toBeInTheDocument();
+      expect(screen.getByRole('tab', { name: 'Versions' })).toHaveClass('euiTab-isSelected');
+      expect(within(screen.getByRole('tabpanel')).getByText('Versions')).toBeInTheDocument();
+    },
+    10 * 1000
```

Comment:
It seems the datagrid component take long time to render, it take more time in github runner.

Reply:
I see, let's keep it so that the tests pass. Just feel it's a bit suspicious for it to take so much much to render, we may need to take a deep look in the future.